.package.resolved

```diff
@@ -60,8 +60,8 @@
         "repositoryURL": "https://github.com/krzyzanowskim/CryptoSwift.git",
         "state": {
           "branch": null,
-          "revision": "8d4f6384e0a8cc41f2005247241dd553963a492a",
-          "version": "1.4.1"
+          "revision": "5669f222e46c8134fb1f399c745fa6882b43532e",
+          "version": "1.3.8"
```

Comment:
this will lead to a compilation error (there is a slight breaking change)

I suggest adding `Package.resolved` and `.package.resolved` to sources triggering tuist build checks

Reply:
What's the breaking change?

Reply:
[this](https://github.com/tuist/tuist/pull/3199/files/185544f2754f552f102ada0324451a1fe6d7e745..5493e0212e9e742082156b4d1c23d140c0dcf2a6#diff-fb243814060ed45f0f15a4362ab53b6e9c7bba22795f0aa0ad6ec548ec5c0d8fR171) used to return an optional but in `1.4.1` it no longer does

Reply:
@pepibumur @fortmarek I would add in this PR the change I have done [here](https://github.com/tuist/tuist/pull/3176/commits/302969a1b373e8a2c9da7dbbc09d6cd24740edd0), as it was causing some failures in CI jobs